roles/cockpit-ui/tasks/main.yml

```diff
@@ -41,37 +41,45 @@
   run_once: true
   when: check_docker_registry_exists.rc == 0
 
-- name: Retrieve docker-registry route
+- name: Retrieve docker-registry json
```

Comment:
I think you can get away with moving `register: docker_registry_route_json` to the create task, and still use the same output. It looks like `state: present` will return the same value as doing a `state: list`